src/cli/clusters_create.rs

```diff
@@ -152,9 +161,40 @@ fn clusters_create(
     let payload =
         serde_json::to_string(&definition).map_err(|e| serialize_error(e.to_string(), span))?;
     client
-        .create_cluster(org_id, project_id, payload, deadline, ctrl_c)
+        .create_cluster(
+            org_id.clone(),
+            project_id.clone(),
+            payload,
+            deadline,
+            ctrl_c.clone(),
+        )
         .map_err(|e| client_error_to_shell_error(e, span))?;
 
+    let cluster_json = client
+        .get_cluster(definition.name(), org_id, project_id, deadline, ctrl_c)
+        .map_err(|e| created_cluster_not_registered(e.message()))?;
+
+    let tls_config =
+        RustTlsConfig::new(true, None).map_err(|e| created_cluster_not_registered(e.message()))?;
+
+    let cluster = RemoteCluster::new(
+        RemoteClusterResources {
+            hostnames: vec![cluster_json.connection_string()],
+            username: "".to_string(),
```

Comment:
Are we not able to get all this info from the definition or something? I'm not sure how much value there is registering the cluster if it's still not actually usable.